### DISCUSSION TITLE
[Incorrect behavior] [3.3.0] Arcropolis doesn't detect bgm files

### DISCUSSION THREAD

**Discussion by BeardedBowser** [Raytwo/ARCropolis#322] [Issues]
Recently updated from 2.0.0 to 3.3.0 and Arcropolis didn't detected the sound files. Tested on 2.0.0 (yes, very outdated, but that's the version where everything works almost perfect for me) and no problem at all. I also ran a conflict search and only found one bgm in it. Thanks in advance for taking the time to read!

**Comment by Raytwo** [comment]
Hi there, are you using the new discovery system or the legacy one?
